invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx

```diff
@@ -67,17 +77,28 @@ export const BoardEditableTitle = memo(({ board, isSelected }: Props) => {
 
   if (!isEditing.isTrue) {
     return (
-      <Text
-        size="sm"
-        fontWeight="semibold"
-        userSelect="none"
-        color={isSelected ? 'base.100' : 'base.300'}
-        onDoubleClick={isEditing.setTrue}
-        cursor="text"
-        minW={16}
-      >
-        {localTitle}
-      </Text>
+      <Flex alignItems="center" gap={1} onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
```

Comment:
This gap is a bit too small.

<img width="149" alt="image" src="https://github.com/user-attachments/assets/da71209f-e932-4d9f-8763-b20355a62e86">

Reply:
```suggestion
      <Flex alignItems="center" gap={3} onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
```